src/http/resource.ts

```diff
@@ -1,69 +1,47 @@
 import { Drash } from "../../mod.ts";
 
 /**
- * @memberof Drash.Http
- * @class Resource
- *
- * @description
- *     This is the base resource class for all resources. All resource classes
- *     must be derived from this class.
+ * This is the base resource class for all resources. All resource classes
+ * must be derived from this class.
  */
 export class Resource implements Drash.Interfaces.Resource {
   /**
-   * @description
-   *     A property to hold the middleware this resource uses.
+   * A property to hold the middleware this resource uses.
    *
-   *     All derived middleware classes MUST define this property as static
-   *     (e.g., static middleware = ["MiddlewareClass"];)
-   *
-   * @property string[] middleware
+   * All derived middleware classes MUST define this property as static
+   * (e.g., static middleware = ["MiddlewareClass"];)
    */
   public middleware: { after_request?: []; before_request?: [] } = {};
 
   /**
-   * @description
-   *     A property to hold the name of this resource. This property is used by
-   *     Drash.Http.Server to help it store resources in its resources property
-   *     by name.
-   *
-   * @property string name
+   * A property to hold the name of this resource. This property is used by
+   * Drash.Http.Server to help it store resources in its resources property
+   * by name.
    */
   public name: string = "";
 
   /**
-   * @description
-   *     A property to hold the paths to access this resource.
-   *
-   *     All derived resource classes MUST define this property as static
-   *     (e.g., static paths = ["path"];)
+   * A property to hold the paths to access this resource.
    *
-   * @property string[] paths
+   * All derived resource classes MUST define this property as static
+   * (e.g., static paths = ["path"];)
    */
   public paths: string[] = [];
 
   public paths_parsed: Drash.Interfaces.ResourcePaths[] = [];
```

Comment:
woops. i forgot to add this docblock when i did the linting stuff

```
A property to hold the expanded versions of the paths. An expanded version of a path looks like the following:
    {
      og_path: "/:id",
      regex_path: "^([^/]+)/?$",
      params: ["id"],
    }
```

Reply:
Where do you want that? On `paths` or `paths_parsed`?

Reply:
Paths parsed